backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -133,12 +136,42 @@ public void testUpdateToken_success() throws Exception {
         // Assert.
         assertEquals("token map status is", inMemoryVirtualQueueRepository.getTokenMap().get(tokenId).getStatus(), updateTokenStatusResponse.getUpdatedToken().getStatus());
     }
+    
+    @Test
+    public void testUpdateShop_success() throws Exception {
+        // Arrange.
+        UUID shopId = addShopToRepository(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
+        addTokenListToShop(shopId);
+        UpdateShopStatusRequest updateShopStatusRequest = new UpdateShopStatusRequest(shopId, ShopStatus.DELETED);
+
+        // Act.
+        UpdateShopStatusResponse updateShopStatusResponse = inMemoryVirtualQueueRepository.updateShop(updateShopStatusRequest);
+
+        // Assert.
+        assertEquals("Shop status is", ShopStatus.DELETED, updateShopStatusResponse.getShop().getStatus());
+    }
+    
+    @Test
+    public void testGetCustomersAhead_success() throws Exception {
+        // Arrange.
+        UUID tokenId = addOneToken();
+        UUID shopId = inMemoryVirtualQueueRepository.getTokenMap().get(tokenId).getShopId();
+        addTokenListToShop(shopId);
+
+        // Act.
+        long customersAhead = inMemoryVirtualQueueRepository.getCustomersAhead(tokenId);
+    
+        // Assert.
+        long expectedCustomersAhead = 1;
```

Comment:
Nit. no need of this extra line. Just directly use 1 below.

Reply:
Done.